examples/vg-specs/area_vertical.vg.json

```diff
@@ -3,6 +3,13 @@
     "description": "Area chart showing weight of cars over time (vertical).",
     "autosize": "pad",
     "padding": 5,
+    "encode": {
+        "update": {
+            "fill": {
+                "value": "transparent"
```

Comment:
Yes. Try `brush.vg.json` but without this block. You'll find that events don't fire if you're interacting in the "whitespace."